templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{object|yesno: "Update, Create"}}Create Manufacturer</h1>
+  <form action="/" method="post" novalidate>
```

Comment:
The form `action` attribute is set to `/`, which might not be the intended URL for handling the form submission. Consider using the appropriate URL for the manufacturer create or update view, such as `{% url 'taxi:manufacturer-create' %}` or `{% url 'taxi:manufacturer-update' pk=object.id %}` depending on the context.